R/run.R

```diff
@@ -0,0 +1,106 @@
+#' run
+#'
+#' Run one or multiple climate models over one or multiple emissions scenarios.
+#'
+#' Some simple climate models need certain information to run. Use the setup function to supply this information
+#' before attempting to use run.
+#'
+#' @param climateModelsConfigs Named list where the names are climate models and the corresponding list member is a
+#' configuration or list of configurations to use for this model. Use null to denote the default configuration with
+#' no changes. Climate model names supported: CiceroSCM, FaIR, and MAGICC7. The supported configuration settings
+#' depend on the used climate model, please refer to their respective documentation.
+#' @param scenarios DataFrame containing one or multiple emissions scenarios to simulate. The data shoud be in
+#' IAMC format, i.e. a wide data frame with the years as columns. Index columns that are additionally required are
+#' "model", "scenario", "region", "variable", and "unit". The "variable" must also follow the IAMC format naming
+#' conventions, e.g. "Emissions|CO2" for the 'total CO2 emissions (not including CCS)'.
+#' @param outputVariables A list of variables to include in the output. Optional, default: list("Surface Temperature")
```

Comment:
```suggestion
#' @param outputVariables A list of variables to include in the output.
```
The default is already shown on the documentation page. Also, this could get out of sync with the actual default value.